src/root.component.tsx

```diff
@@ -17,6 +18,10 @@ function Root(props) {
         component={PatientChartSummary}
       />
       <Route path="/patient/:patientUuid/chart" component={LevelTwoRoutes} />
+      <Route
```

Comment:
I think we should be centralizing our routes. I don't think having routes all over the place is easy to manage. @fatmali what do you think?

Reply:
We can have the level two and three routes in the same file as a variable instead of an import, the nesting helps routes to be more consistent, so I suggest we stick with the structure and just refactor.